mmpose/models/losses/multi_loss_factory.py

```diff
@@ -9,20 +9,19 @@
 from ..registry import LOSSES
 
 
-def _make_input(t, requires_grad=False, need_cuda=True):
+def _make_input(t, requires_grad=False, device=torch.device('cpu')):
     """Make zero inputs for AE loss.
 
     Args:
         t (torch.Tensor): input
         requires_grad (bool): Option to use requires_grad.
-        need_cuda (bool): Opthin to use cuda.
+        device: torch device
     Returns:
         inp (torch.Tensor): zero input.
     """
     inp = torch.autograd.Variable(t, requires_grad=requires_grad)
```

Comment:
Unrelated to this pr, but Variable has been deprecated. https://pytorch.org/docs/stable/autograd.html#variable-deprecated

Reply:
Maybe we can remove all usage of Variables in a separate PR.

Reply:
Good idea.